README.md

```diff
@@ -108,6 +108,32 @@ module.exports = {
 }
 ```
 
+### Disabling HMR
+
+By default Hot Module Replacement is disabled in following situations:
+
+ * Webpack `target` is `node`
+ * Webpack doesn't minify the code 
+ * `process.env.NODE_ENV !== 'production'`
```

Comment:
In the sentence above the list I wrote: By default Hot Module Replacement is **disabled** 

Should I change the sentence to be positive, like:
By default Hot Module Replacement is **enabled** when:
* Webpack `target` is not `node`
* Webpack minifies the code
* `process.env.NODE_ENV === 'production'`